dataprofiler/profilers/profiler_options.py

```diff
@@ -991,15 +1004,27 @@ def _validate_helper(self, variable_path='StructuredOptions'):
             ('data_labeler', DataLabelerOptions),
             ('correlation', CorrelationOptions)
         ])
-
-        for column in self.properties:
+        properties = self.properties
+        properties.pop('null_values')
+        for column in (properties):
             if not isinstance(self.properties[column], prop_check[column]):
                 errors.append("{}.{} must be a(n) {}.".format(
                     variable_path, column, prop_check[column].__name__))
             else:
                 errors += self.properties[column]._validate_helper(
                     variable_path=(variable_path + '.' + column
                                    if variable_path else column))
+
+        if self.null_values is not None and not (
+                isinstance(self.null_values, dict) and
+                all(isinstance(key, str) and
+                    (isinstance(value, re.RegexFlag) or value == 0)
```

Comment:
is value necessary 0 or it can be NO_FLAG in general?

Reply:
`NO_FLAG == 0`